components/jaeger-collector.libsonnet

```diff
@@ -75,8 +98,11 @@ local service = k.core.v1.service;
         ) +
         container.withVolumeMounts([mount.new('jaeger-store-data', '/var/jaeger/store')]) +
         container.mixin.readinessProbe.withFailureThreshold(3) +
+        container.mixin.readinessProbe.withPeriodSeconds(30) +
+        container.mixin.readinessProbe.withInitialDelaySeconds(30) +
```

Comment:
Are you really getting up to 30s for the Jaeger service to start? That's an awful lot of time :)

Reply:
🙈 Just my default values, I'll fix them.

Reply:
10s should be more than enough, but not sure how long Jaeger needs to start when using badger with a big pre-existing data set. How long is Jaeger typically taking to start in your production instances?

Reply:
We haven't measured that, I'll have a look.